src/NoTimeToWaitBundle/Resources/views/Default/nttw5.html.twig

```diff
@@ -31,19 +31,18 @@
 <p>The conference will be livestreaming on the MediaArea YouTube channel <a href="http://www.youtube.com/c/MediaAreaNetOfficial">available here</a>.</p>
 
 <h2>Where</h2>
-<p><a href="www.beeldengeluid.nl">The Netherlands Institute for Sound and Vision</a>, Media Parkboulevard 1, 1217 WE Hilversum, Netherlands</p>
+<p><a href="//www.beeldengeluid.nl">The Netherlands Institute for Sound and Vision</a>, Media Parkboulevard 1, 1217 WE Hilversum, Netherlands.</p>
```

Comment:
`https://www.beeldengeluid.nl/` ?

Reply:
[Protocol-relative URL](https://en.wikipedia.org/wiki/URL#prurl), I use to let the user keeping the protocol used on the page even if nowadays lot of people advocate for switching to all in HTTPS. And trailing slash for root page is less used nowadays (hidden in browsers...).

Reply:
Thank you for the information! I didn’t know that. Yet why not for the other links as well?

Reply:
Some incoherencies with different contributors (and I also sometimes forget to do it), I need to normalize that at some point.

Reply:
I can open a PR once this one has been merged.